docs/presentation/Content/ContentRelatedInstances.md

```diff
@@ -38,13 +38,13 @@ Either a single or an array of specifications may be specified for this attribut
 Specifies an [ECExpression](./ECExpressions.md#instance-filter) for filtering instances of ECClasses targeted through the [`relationshipPaths` attribute](#attribute-relationshippaths).
 
 ```ts
-[[include:SharedAttributes.InstanceFilter.Ruleset]]
+[[include:ContentRelatedInstances.InstanceFilter.Ruleset]]
 ```
 
-  |                | Result                                                                                                       |
-  | -------------- | ------------------------------------------------------------------------------------------------------------ |
-  | without filter | ![Example when selecting all instances](./media/contentinstancesofspecificclasses-with-instancefilter-1.png) |
-  | with filter    | ![Example when filtering instances](./media/contentinstancesofspecificclasses-with-instancefilter-2.png)     |
+  |                | Result                                                                                      |
+  | -------------- | ------------------------------------------------------------------------------------------- |
+  | without filter | ![Example when selecting all instances](./media/sharedattributes-with-instancefilter-1.png) |
+  | with filter    | ![Example when filtering instances](./media/sharedattributes-with-instancefilter-2.png)     |
```

Comment:
Does the image really match what the ruleset returns for the iModel?

Reply:
There is a ContentModifier applies to reduce the amount of properties. In this case it's mainly set to show the pitch property
```json
{
	"ruleType": "ContentModifier",
	"class": { "schemaName": "BisCore", "className": "SpatialViewDefinition" },
	"propertyOverrides": [{
	"name": "Pitch",
	"isDisplayed": true
	}]
}
```